src/resolvers.ts

```diff
@@ -53,5 +53,40 @@ export const resolvers: ResolverMap = {
       console.warn(`Could not load value ${argument} from credstash: ${error}`);
       return undefined;
     });
+  },
+  aws: async (argument: string) => {
```

Comment:
I would probably not call this “aws”, since it’s such a ubiquitous term.  Maybe “asm” for AWS Secret Manager?

Reply:
Agree - Went with `asm`